### PR TITLE
Add doc for importing and unmanaging DATA volumes

### PR DESCRIPTION
**Pull request by weizhouapache**
this is the document for [apache/cloudstack#8808](https://github.com/apache/cloudstack/pull/8808)

----
📚 Documentation preview 📚: https://cloudstack-documentation--390.org.readthedocs.build/en/390/

### CODE REVIEW

**Comment by weizhouapache** [comment]
https://cloudstack-documentation--390.org.readthedocs.build/en/390/adminguide/storage.html#importing-and-unmanaging-volumes-from-storage-pools

**Review by kiranchavala — COMMENTED** [review]
@weizhouapache  can we mention that only qcow2 format data volumes are supported for import and  raw and other image formats are not supported.

The format of qcow2 can be checked by executing the command 

qemu-img info <>

--
Also, can we mention encrypted volumes  are not supported

**Comment by weizhouapache** [comment]
> @weizhouapache can we mention that only qcow2 format data volumes are supported for import and raw and other image formats are not supported.
> 
> The format of qcow2 can be checked by executing the command
> 
> qemu-img info <>
> 
> -- Also, can we mention encrypted volumes are not supported

thanks @kiranchavala 
I will update

**Comment by weizhouapache** [comment]
> @weizhouapache can we mention that only qcow2 format data volumes are supported for import and raw and other image formats are not supported.
> 
> The format of qcow2 can be checked by executing the command
> 
> qemu-img info <>
> 
> -- Also, can we mention encrypted volumes are not supported

added to notes
thanks @kiranchavala

**Review by kiranchavala — APPROVED** [review]
LGTM, the doc for import data volumes looks good

File: source/adminguide/storage.rst [hunk 1]
```diff
@@ -1011,6 +1011,60 @@ Following is the example for checkVolume API usage and the result in the volume
...
+.. note::
+   The volume to be imported must be placed in the root directory of the storage pool.
+   The format of the volume should be QCOW2 on NFS/Local storage, and RAW on Ceph storage.
+   Encrypted volumes cannot be unmanaged or imported.
+
+.. note::
+   By default, the volume is imported for the caller if Domain/Account/Project are not set. By default, the imported volumes use the default disk offering "Default Custom Offering for Volume Import"
+   (on Shared storages) or "Default Custom Offering for Volume Import - Local" (on Local storages).
+
+To unmanage volume(s),
```
Comment (andrijapanicsb) [inline source/adminguide/storage.rst:1058]:
```suggestion
To unmanage volume(s): 
```

File: source/adminguide/storage.rst [hunk 2]
```diff
@@ -1011,6 +1011,60 @@ Following is the example for checkVolume API usage and the result in the volume
...
+
+#. Select the storage pool.
+
+#. The unmanaged volumes on the storage pool are listed on the left panel.
+
+#. The managed volumes on the storage pool are listed on the right panel.
+
+|list-unmanaged-managed-volumes.png|
+
+To import a volume,
```
Comment (andrijapanicsb) [inline source/adminguide/storage.rst:1039]:
```suggestion
To import a volume: 
```

File: source/adminguide/storage.rst [hunk 3]
```diff
@@ -1011,6 +1011,60 @@ Following is the example for checkVolume API usage and the result in the volume
...
+#. In the Import Volume pop-up, select the Account Type, Domain/Account/Project and a disk offering.
+
+#. Click OK.
+
+|import-volume.png|
+
+.. note::
+   The volume to be imported must be placed in the root directory of the storage pool.
+   The format of the volume should be QCOW2 on NFS/Local storage, and RAW on Ceph storage.
+   Encrypted volumes cannot be unmanaged or imported.
```
Comment (weizhouapache) [inline source/adminguide/storage.rst:1052]:
```suggestion
   The following volumes cannot be unmanaged or imported: (1) volume is encrypted; (2) volume is locked; (3) volume has a backing file.
```

**Pull request merged**